bootstrap/test/clean.test.ts

```diff
@@ -0,0 +1,11 @@
+import {} from 'jest';
+import * as childProcess from 'child_process';
+import * as fs from 'fs';
+
+test('GET /test', async() => {
+    childProcess.spawnSync('./absolute', ['build_server']);
```

Comment:
Could you try childProcess.spawnSync('./absolute', ['build_server'], { shell: true });?